skrifa/src/outline/autohint/latin/blues.rs

```diff
@@ -436,6 +435,153 @@ fn compute_default_blues(font: &FontRef, coords: &[F2Dot14], style: &StyleClass)
     blues
 }
 
+/// Compute unscaled blue values for the CJK script set.
+///
+/// See <https://gitlab.freedesktop.org/freetype/freetype/-/blob/57617782464411201ce7bbc93b086c1b4d7d84a5/src/autofit/afcjk.c#L277>
+fn compute_cjk_blues(font: &FontRef, coords: &[F2Dot14], style: &StyleClass) -> [UnscaledBlues; 2] {
+    let mut blues = [UnscaledBlues::new(), UnscaledBlues::new()];
+    let mut outline_buf = UnscaledOutlineBuf::<MAX_INLINE_POINTS>::new();
+    let mut flats = [0; BLUE_STRING_MAX_LEN];
+    let mut fills = [0; BLUE_STRING_MAX_LEN];
+    let glyphs = font.outline_glyphs();
+    let charmap = font.charmap();
+    let units_per_em = font
+        .head()
+        .map(|head| head.units_per_em())
+        .unwrap_or_default() as i32;
```

Comment:
as with buffers this seems like it could be hoisted out. `let (glyphs, charmap, units_per_em) = things_all_blues_need(font);` (we must get our Dr Seuss refs in; https://seuss.fandom.com/wiki/Thneed)